### DISCUSSION TITLE
Can we please have regular bugfix releases?

### DISCUSSION THREAD

**Discussion by witcheslive** [transmission/transmission#2731] [General]
the bug where adding torrents randomly crashes the client has been driving me insane for well over a year now and the last release was two years ago. the bug has apparently been fixed for several months by now. it seems like a lot of hard work is done and active development, and thank you! as such it would be amazing if it was actually available for the bulk of users and use cases. could we please have some minor releases so we can use it without manually compiling and installing side versions and can just update normally? Even if it's only every month or every other month, it would be very nice.

**Comment by limburgher** [comment]
Agreed, in fact I need one so that Transmission will work on Fedora 36.
https://github.com/transmission/transmission/issues/2126
